cmd_getprop.c

```diff
@@ -42,9 +42,24 @@ getprop_main(const struct cmd_getprop_info* info)
 // become inefficient when Android 4.4 switched to a
 // trie implementation.
 
+static void* libc;
 static int (*property_foreach)(
         void (*propfn)(const prop_info *pi, void *cookie),
         void *cookie);
+static const prop_info* (*property_find_nth)(unsigned n);
+
+static void find_symbol_in_libc(const char* name, void* fnptr_v)
+{
+    void** fnptrptr = fnptr_v;
```

Comment:
nit: why not just type `fnptr_v` as it actually is; `void**`? i know there's no real difference between `void*` and `void**` but this line was a source of confusion for a minute.